src/DataTypes/Serializations/tests/gtest_qbit_serialization.cpp

```diff
@@ -4,29 +4,35 @@
 #include <Formats/FormatSettings.h>
 #include <IO/ReadBufferFromString.h>
 #include <IO/WriteBufferFromString.h>
+#include <Common/ErrorCodes.h>
 
 #include <base/types.h>
 #include <gtest/gtest.h>
 
 
-using namespace DB;
+#define EXPECT_THROW_ERROR_CODE(statement, expected_exception, expected_code) \
```

Comment:
Maybe we should move the macro from [here](https://github.com/clickhouse/clickhouse/blob/master/src/IO/tests/gtest_memory_resize.cpp#L6-L17) to some header?